ironfish/src/rpc/adapters/socketAdapter/socketAdapter.ts

```diff
@@ -35,6 +36,8 @@ export abstract class RpcSocketAdapter implements IRpcAdapter {
   server: net.Server | null = null
   router: Router | null = null
   namespaces: ApiNamespace[]
+  rpcAuthTokenPath: string
+  fileSystem: FileSystem
```

Comment:
I think we should move this auth level up to the RpcServer, so we don't have to do this in every adapter we want to implement auth for.